### PR TITLE
Hopefully I fixed everything... 

### PR DESCRIPTION
**Pull request by RedPrez16**

### CODE REVIEW

File: websites/B/Bungie/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "author": {
+    "name": "RedBird16.com",
+    "id": "444872133504663552"
+  },
+  "service": "Bungie",
+  "description": {
+    "en": "Bungie.net is the home for all your Destiny/Destiny Two needs!"
+  },
+  "url": ["www.bungie.net", "bungie.net", "comics.bungie.net", "bungiestore.com"],
+  "version": "One",
```
Comment (Bas950) [inline websites/B/Bungie/dist/metadata.json:11]: version 1.0.0

File: websites/B/Bungie/presence.ts [hunk 2]
```diff
@@ -0,0 +1,160 @@
+var presence = new Presence({
+    clientId: "711393222252822539"
+});
+var browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (Bas950) [inline websites/B/Bungie/presence.ts:4]: Dont use var, use const

File: websites/B/Bungie/dist/presence.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
...YSxDQUFDO1FBQzVDLFlBQVksQ0FBQyxPQUFPLEdBQUcsQ0FBQyxVQUFVLEdBQUMsY0FBYyxDQUFDLFNBQVMsQ0FBQyxDQUFBO0tBQzdEO1NBQ0ksSUFBSSxJQUFJLENBQUMsUUFBUSxDQUFFLG9CQUFvQixDQUFDLEVBQUM7UUFDNUMsSUFBSSxhQUFhLEdBQUcsUUFBUSxDQUFDLGFBQWEsQ0FBQyxvQ0FBb0MsQ0FBQyxDQUFBO1FBQ2hGLFlBQVksQ0FBQyxjQUFjLEdBQUcsYUFBYSxDQUFDO1FBQzVDLFlBQVksQ0FBQyxPQUFPLEdBQUcsQ0FBQyxVQUFVLEdBQUMsYUFBYSxDQUFDLFNBQVMsQ0FBQyxDQUFBO0tBQzVEO1NBQ0ksSUFBSSxJQUFJLENBQUMsUUFBUSxDQUFFLHVCQUF1QixDQUFDLEVBQUM7UUFDaEQsSUFBSSxZQUFZLEdBQUcsUUFBUSxDQUFDLGFBQWEsQ0FBQyxtS0FBbUssQ0FBQyxDQUFBO1FBQzdNLFlBQVksQ0FBQyxjQUFjLEdBQUcsYUFBYSxDQUFDO1FBQzVDLFlBQVksQ0FBQyxPQUFPLEdBQUcsQ0FBQyxVQUFVLEdBQUMsWUFBWSxDQUFDLFNBQVMsQ0FBQyxDQUFBO0tBQzNEO1NBQ0ksSUFBSyxJQUFJLENBQUMsUUFBUSxDQUFFLDBCQUEwQixDQUFDLEVBQUM7UUFDbkQsSUFBSSxXQUFXLEdBQUcsUUFBUSxDQUFDLGFBQWEsQ0FBQyxtS0FBbUssQ0FBQyxDQUFBO1FBQy9NLFlBQVksQ0FBQyxjQUFjLEdBQUcsYUFBYSxDQUFDO1FBQzVDLFlBQVksQ0FBQyxPQUFPLEdBQUcsQ0FBQyxVQUFVLEdBQUMsV0FBVyxDQUFDLFNBQVMsQ0FBQyxDQUFBO0tBQ3hEO0lBQ0QsSUFBSSxZQUFZLENBQUMsT0FBTyxJQUFJLElBQUksRUFBRTtRQUNoQyxRQUFRLENBQUMsWUFBWSxFQUFFLENBQUM7UUFDeEIsUUFBUSxDQUFDLFdBQVcsRUFBRSxDQUFDO0tBQ3hCO1NBQ0k7UUFDSCxRQUFRLENBQUMsV0FBVyxDQUFDLFlBQVksQ0FBQyxDQUFDO0tBQ3BDO0FBQUEsQ0FBQyxDQUFDLENBQUEifQ==
```
Comment (Bas950) [inline websites/B/Bungie/dist/presence.js:160]: you are not allowed to push js files

File: websites/B/Bungie/presence.ts [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
...YSxDQUFDO1FBQzVDLFlBQVksQ0FBQyxPQUFPLEdBQUcsQ0FBQyxVQUFVLEdBQUMsY0FBYyxDQUFDLFNBQVMsQ0FBQyxDQUFBO0tBQzdEO1NBQ0ksSUFBSSxJQUFJLENBQUMsUUFBUSxDQUFFLG9CQUFvQixDQUFDLEVBQUM7UUFDNUMsSUFBSSxhQUFhLEdBQUcsUUFBUSxDQUFDLGFBQWEsQ0FBQyxvQ0FBb0MsQ0FBQyxDQUFBO1FBQ2hGLFlBQVksQ0FBQyxjQUFjLEdBQUcsYUFBYSxDQUFDO1FBQzVDLFlBQVksQ0FBQyxPQUFPLEdBQUcsQ0FBQyxVQUFVLEdBQUMsYUFBYSxDQUFDLFNBQVMsQ0FBQyxDQUFBO0tBQzVEO1NBQ0ksSUFBSSxJQUFJLENBQUMsUUFBUSxDQUFFLHVCQUF1QixDQUFDLEVBQUM7UUFDaEQsSUFBSSxZQUFZLEdBQUcsUUFBUSxDQUFDLGFBQWEsQ0FBQyxtS0FBbUssQ0FBQyxDQUFBO1FBQzdNLFlBQVksQ0FBQyxjQUFjLEdBQUcsYUFBYSxDQUFDO1FBQzVDLFlBQVksQ0FBQyxPQUFPLEdBQUcsQ0FBQyxVQUFVLEdBQUMsWUFBWSxDQUFDLFNBQVMsQ0FBQyxDQUFBO0tBQzNEO1NBQ0ksSUFBSyxJQUFJLENBQUMsUUFBUSxDQUFFLDBCQUEwQixDQUFDLEVBQUM7UUFDbkQsSUFBSSxXQUFXLEdBQUcsUUFBUSxDQUFDLGFBQWEsQ0FBQyxtS0FBbUssQ0FBQyxDQUFBO1FBQy9NLFlBQVksQ0FBQyxjQUFjLEdBQUcsYUFBYSxDQUFDO1FBQzVDLFlBQVksQ0FBQyxPQUFPLEdBQUcsQ0FBQyxVQUFVLEdBQUMsV0FBVyxDQUFDLFNBQVMsQ0FBQyxDQUFBO0tBQ3hEO0lBQ0QsSUFBSSxZQUFZLENBQUMsT0FBTyxJQUFJLElBQUksRUFBRTtRQUNoQyxRQUFRLENBQUMsWUFBWSxFQUFFLENBQUM7UUFDeEIsUUFBUSxDQUFDLFdBQVcsRUFBRSxDQUFDO0tBQ3hCO1NBQ0k7UUFDSCxRQUFRLENBQUMsV0FBVyxDQUFDLFlBQVksQ0FBQyxDQUFDO0tBQ3BDO0FBQUEsQ0FBQyxDQUFDLENBQUEifQ==
```
Comment (Bas950) [inline websites/B/Bungie/presence.ts:160]: there is in no way possible to have a sourcemapping in a ts file. So funny how you manage to do this.

File: websites/B/Bungie/dist/presence.js (hunk 3 above)
Comment (RedPrez16) [reply websites/B/Bungie/dist/presence.js:160]: I didn't? If so, I'm sorry. Is there a way to remove it?

File: websites/B/Bungie/presence.ts (hunk 4 above)
Comment (RedPrez16) [reply websites/B/Bungie/presence.ts:160]: Guess what, it just did it on it's own.

**Comment by Alanexei** [comment]
Upload screenshots showing a side-by-side comparison of your Discord presence working and the page it is on.

**Comment by Alanexei** [comment]
Look at other pull requests to get ideas.

**Comment by RedPrez16** [comment]
> 
> 
> Upload screenshots showing a side-by-side comparison of your Discord presence working and the page it is on.

https://Imgur.com/a/El1Dutg

I hope an Imgur link is okay.

**Comment by RedPrez16** [comment]
> 
> 
> Fix spelling mistakes

Where? I didn't see any...

**Comment by Alanexei** [comment]
Look at your code

**Comment by RedPrez16** [comment]
> 
> 
> Look at your code

Wow, you're so helpful...

**Comment by Alanexei** [comment]
I am not supposed to spoon feed you locations, that is for you to solve, not me.

**Comment by RedPrez16** [comment]
> 
> 
> I am not supposed to spoon feed you locations, that is for you to solve, not me.

I just don't see any period...

**Comment by RedPrez16** [comment]
> 
> 
> I am not supposed to spoon feed you locations, that is for you to solve, not me.

Also, it's funny that I'm being "graded" by a person that's younger than me...

**Comment by Alanexei** [comment]
This is your final warning, if you don't fix the things that need to be fixed, I am closing this pull request.

File: websites/B/Bungie/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    "id": "444872133504663552"
+  },
+  "service": "Bungie",
+  "description": {
+    "en": "Bungie.net is the home for all your Destiny/Destiny Two needs!"
+  },
+  "url": ["www.bungie.net", "bungie.net", "comics.bungie.net", "bungiestore.com"],
+  "version": "1.0.0",
+"logo": "https://i.imgur.com/7N4tOFk.jpg",
+"thumbnail": "https://imgur.com/a/szETlTp",
```
Comment (doomlerd) [inline websites/B/Bungie/dist/metadata.json:14]: Link should be an image, not an album

Comment (RedPrez16) [reply websites/B/Bungie/dist/metadata.json:14]: Yep, you're right. I forgot about that. I'm sorry!

**Comment by RedPrez16** [comment]
> 
> 
> Use textContent/innerText instead of innerHTML as they're more practical, and might as well reduce the amount of conditions using a switch statement.

I would, but that doesn't work... For this website at least...

**Comment by doomlerd** [comment]
Closing in 3 days, you only have to change the thumbnail's URL though, the rest is optional.

**Pull request merged**